GitCommands/Remote/GitRemoteManager.cs

```diff
@@ -68,20 +68,27 @@ public GitRemoteManager(Func<IGitModule> getModule)
         [SuppressMessage("ReSharper", "RedundantArgumentDefaultValue")]
         public void ConfigureRemotes(string remoteName)
         {
-            var module = GetModule();
-            var localConfig = module.LocalConfigFile;
+            var remoteNameToUpper = remoteName.ToUpperInvariant();
+
+            var _module = GetModule();
```

Comment:
On `master` this name will have been changed to remove the leading `_`. Such identifiers are now compile errors (hence need to rebase).